385/Assets/Scripts/PlayerCamera.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+using Assets.Scripts;
+
+public class PlayerCamera : MonoBehaviour {
+    // player object camera is attached to
+    public GameObject player;
+
+    // offset of camera from player
+    private Vector3 offset;
+
+    // Static position of camera
+    private Vector3 initPos;
+
+    // camera that follows player
+    public Camera playerCam;
+
+    // checks Camera1 to change camera settings
+    public string CameraAxis = "Camera1";
+
+    // used to check is camera button is pushed
+    private AxisButton CameraButton;
+
+    public bool trackPlayer = false;
+
+    public float moveSpeed = 1;
+
+    public float distance = 2, X; // controls distance of camera and 
```

Comment:
I think that some of these variables can have more descriptive names. In addition, why is there only an X offset, and not an entire `Vector3` for the camera offset.

Reply:
variable names adjusted, camYoffset made public and Vector3 offset for adjusting cam offset made public

Reply:
What purpose does the camYoffset serve that the Vector3 offset doesn't? Vector3 has an X Y and a Z.